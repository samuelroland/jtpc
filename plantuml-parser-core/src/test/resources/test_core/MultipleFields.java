public class MultipleFields {
    private int x, y; protected String hello, bye, greet;
}
