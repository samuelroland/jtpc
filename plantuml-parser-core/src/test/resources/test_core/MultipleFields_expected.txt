@startuml
class MultipleFields {
	- x: int
	- y: int
	# hello: String
	# bye: String
	# greet: String
}
@enduml
